### ISSUE TITLE
Pagination does not work correctly [Android]

### ISSUE THREAD

**Issue by Obi1Kennoby** [react-native-community/react-native-cameraroll#118]
# Bug

I have around 1000 photos on my phone and when I try to load them by chunks (15 photos per time) only 109 photo loaded but if set chunk size to 200, I be able to load 325 photos

## Environment info
System:
    OS: macOS 10.15.1
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
    Memory: 32.89 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.12.0 - /usr/local/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 6.12.0 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 26, 27, 28
      Build Tools: 27.0.3, 28.0.0, 28.0.2, 28.0.3
      System Images: android-26 | Google APIs Intel x86 Atom, android-26 | Google Play Intel x86 Atom, android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
      Android NDK: 14.1.3816874
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 11.2.1/11B500 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5 
  npmGlobalPackages:
    react-native-git-upgrade: 0.2.7
  Device: Samsung Galaxy s10e, android 10

Library version: 1.3.0

Describe what you expected to happen:
It should be possible to load all available photos on a phone

## Reproducible sample code
```typescript
const IMAGES_PER_PAGE = 15

interface CurrentPageInfo {
  has_next_page: boolean // eslint-disable-line
  end_cursor?: string // eslint-disable-line
}

interface IImage {
  id?: string
  path: string
  mime: string
}

interface State {
  library: any[]
  results: IImage[]
}

export default class ImagePickerScreen extends Component<{}, State> {
  state: State = {
    library: [],
    results: [],
  }

  isFetchingPhotos = false
  pagesFetched = 0
  currentPageInfo: CurrentPageInfo = { has_next_page: true }

  componentDidMount() {
    this.fetchNextPhotos()
  }

  fetchNextPhotos = async () => {
    if (isAndroid) {
      const permission = await PermissionsAndroid.check(
        PermissionsAndroid.PERMISSIONS.READ_EXTERNAL_STORAGE,
      )

      if (!permission) {
        const granted = await PermissionsAndroid.request(
          PermissionsAndroid.PERMISSIONS.READ_EXTERNAL_STORAGE,
        )
        if (granted !== PermissionsAndroid.RESULTS.GRANTED) return
      }
    }

    if (this.isFetchingPhotos || !this.currentPageInfo.has_next_page) {
      return
    }

    this.isFetchingPhotos = true

    try {
      const cameraRollParams: GetPhotosParamType = {
        first: IMAGES_PER_PAGE,
        after: this.currentPageInfo.end_cursor,
        assetType: 'Photos',
      }
      if (isIos) {
        cameraRollParams.groupTypes = 'All'
      }

      const data = await CameraRoll.getPhotos(cameraRollParams)
      this.setState({
        library: [...this.state.library, ...data.edges.map(edge => edge.node.image)],
      })
      this.pagesFetched++
      this.currentPageInfo = data.page_info
    } catch (e) {
      postBugToBugsee('CameraRoll get photos error', e.message)
    } finally {
      this.isFetchingPhotos = false
    }
  }

  // I simplified code for example,
  // originally I delete duplicate images,
  // so it't not a problem to use uri as a key
  keyExtractor = item => item.uri + Math.random() * 10000

  renderThumbnail = ({ item, index }) => {
    return (
      <Image
        source={{ uri: item.uri }}
        style={{
          width: 100,
          height: 100,
          marginBottom: 3,
        }}
      />
    )
  }

  render() {
    return (
      <FlatList
        numColumns={3}
        columnWrapperStyle={{ justifyContent: 'space-between' }}
        data={this.state.library}
        extraData={this.state.results}
        keyExtractor={this.keyExtractor}
        renderItem={this.renderThumbnail}
        onEndReached={this.fetchNextPhotos}
      />
    )
  }
}
```

**Comment by thekevinbrown** [comment]
I wonder if this is related to #47.

**Comment by thekevinbrown** [comment]
Having looked at it, I think this is very much a duplicate of #47. I don't have the ability to close this issue, but perhaps it'd be best if we tracked the conversation there.

**Comment by YasirNaeemHw** [comment]
Issue resolve after update of package 5.6.0
